spikingjelly/activation_based/triton_kernel/flexsn/__init__.py

```
@@ -1,11 +1,15 @@
 try:
     from .info import *
```

Ruff (0.15.12)

[error] `from .info import *` used; unable to detect undefined names

(F403)